src/coordinator.rs

```diff
@@ -148,7 +162,7 @@ impl Builder {
         let file_hashes = File::create(file_hashes_path)
             .context("could not open file hash cache to store new hashes")?;
         // TODO: BufWriter?
-        serde_json::to_writer(file_hashes, &coordinator.meta_to_hash)
+        serde_json::to_writer(file_hashes, &meta_to_hash)
```

Comment:
Does it matter that this order could theoretically change? Like to meta_to_hash with the same items but different order.

Reply:
nah, this one's fine. It's just used as a KV store. Another one of those places that'll move to a better database eventually.